### PR TITLE
Misc.

### PR DESCRIPTION
**Pull request by Helenerineium**
Update opus git
Fix shaderc install

### CODE REVIEW

**Comment by wiiaboo** [comment]
"Misc" is not a proper commit message and that's a lot of unrelated changes in one single commit.
Please rebase it with relevant changes split by functionality, i.e. all git URL changes to xiph could be made in a single commit, but nothing else.

Don't change cmake `no` to `off` unless it's broken. That's just unnecessary changes for what's a synonym.

File: README.md [hunk 1]
```diff
@@ -30,9 +30,9 @@ For information about the compiler environment see the wiki, there you also have
         - libx265 (hg)
         - nvenc (built-in)
         - schannel with gmp (mingw)
-            - enabled by default if openssl, libtls or gnutls aren't enabled
+            - enabled by default if openssl, libtls, mbedtls or gnutls aren't enabled
```
Comment (wiiaboo) [inline README.md]: All these changes in the readme that aren't due to changes in the suite need to be a split commit

File: build/media-suite_compile.sh [hunk 2]
```diff
@@ -599,7 +599,7 @@ fi
 unset _deps
 
 _check=(libopus.{,l}a opus.pc opus/opus.h)
-if enabled libopus && do_vcs "https://github.com/xiph/opus.git"; then
+if enabled libopus && do_vcs "https://git.xiph.org/opus.git"; then
```
Comment (wiiaboo) [inline build/media-suite_compile.sh]: Split these git URL changes to a separate commit (for all of them to git.xiph.org).

File: build/media-suite_compile.sh [hunk 3]
```diff
@@ -647,7 +647,7 @@ if [[ $ffmpeg != "no" ]] && enabled libsoxr; then
     _check=(soxr.h libsoxr.a)
     if do_vcs https://notabug.org/RiCON/soxr.git libsoxr; then
         do_uninstall "${_check[@]}"
-        do_cmakeinstall -DWITH_LSR_BINDINGS=off -DBUILD_TESTS=off -DWITH_OPENMP=off
+        do_cmakeinstall -DWITH_LSR_BINDINGS=OFF -DBUILD_TESTS=OFF -DWITH_OPENMP=OFF
```
Comment (wiiaboo) [inline build/media-suite_compile.sh]: Why?

File: build/media-suite_compile.sh [hunk 4]
```diff
@@ -1560,16 +1560,15 @@ if [[ $vvc = y ]] &&
     do_patch "https://0x0.st/sG0V.txt"
     _notrequired="true"
     # install to own dir because the binaries' names are too generic
-    do_cmakeinstall -DCMAKE_INSTALL_BINDIR="$LOCALDESTDIR"/bin-video/vvc \
-        -DBUILD_STATIC=on
+    do_cmakeinstall -DCMAKE_INSTALL_BINDIR="$LOCALDESTDIR"/bin-video/vvc -DBUILD_STATIC=ON
```
Comment (wiiaboo) [inline build/media-suite_compile.sh]: Sure, but why?

File: build/media-suite_compile.sh [hunk 5]
```diff
@@ -1560,16 +1560,15 @@ if [[ $vvc = y ]] &&
...
     do_checkIfExist
     unset _notrequired
 fi
 
 enabled openssl && hide_libressl
 if [[ $ffmpeg != "no" ]]; then
     enabled libgsm && do_pacman_install gsm
-    enabled libsnappy && do_addOption --extra-libs=-lstdc++ && do_pacman_install snappy
+    enabled libsnappy && do_pacman_install snappy
```
Comment (wiiaboo) [inline build/media-suite_compile.sh]: Why?

File: build/media-suite_compile.sh [hunk 6]
```diff
@@ -1894,11 +1893,12 @@ if [[ $mpv != "n" ]] && pc_exists libavcodec libavformat libswscale libavfilter;
         add_third_party "https://github.com/KhronosGroup/SPIRV-Cross.git" spirv-cross
 
         # fix python indentation errors from non-existant code review
-        grep -ZRlP --include="*.py" '\t' third_party/spirv-tools/ | xargs -r -0 -n1 sed -i 's;\t;    ;g'
+        # grep -ZRlP --include="*.py" '\t' third_party/spirv-tools/ | xargs -r -0 -n1 sed -i 's;\t;    ;g'
```
Comment (wiiaboo) [inline build/media-suite_compile.sh]: Why? This is to prevent issues from the committing broken python files as it has happened before.

File: build/media-suite_compile.sh [hunk 7]
```diff
@@ -1894,11 +1893,12 @@ if [[ $mpv != "n" ]] && pc_exists libavcodec libavformat libswscale libavfilter;
...
 
-        do_cmake -GNinja -DSHADERC_SKIP_TESTS=ON
+        do_cmake -DSHADERC_SKIP_TESTS=ON -DSHADERC_SKIP_INSTALL=ON
```
Comment (wiiaboo) [inline build/media-suite_compile.sh]: Why did you remove the -GNinja? It's used by default by `do_cmake` now, but at some points it's not.

**Comment by Helenerineium** [comment]
Sure. Will open another two with proper messages.
